cds-snc/cppd-medical-report-sails
# Dashboard page

Implemented a dashboard page to allow medical professionals to enter the information in any order. 

https://trello.com/c/eYk0GRz8/479-implement-medical-professionals-are-able-to-complete-the-medical-report-in-any-order

## Test

1. Click on "Lets Go" from the start page and it should bring you to the dashboard
1. Fill out any of the following sections: 
    - Basic Patient Information
    - Medical Conditions
    - Medications
    - Treatments
    - Supporting Documents
    - Your information and Declaration
1. The value *Not Completed* should change to *Completed* when the section is filled out properly

## Please Note: 

The following things **are not implemented**

- Consent link for PDF
- Claim 85$ compensation link 
- Save and return later
- Submit to service Canada
- Functional Limitations section
- Future Work Capacity
- Supporting Documents
